src/libraries/Delegator.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/libraries/DelegationTarget.sol';
+
+
+contract Delegator is DelegationTarget {
```

Comment:
Hmm, this shouldn't be showing up in this PR.  I suspect you might have some merge/rebase issues in here.  Let me know if you want some help resolving them.